interval/interval.go

```diff
@@ -43,8 +55,15 @@ func (i *Interval) RandomInstant() time.Time {
 	end := i.End.Unix()
```

Comment:
(this is about line 50 but I can't attach a comment there ?!)

`if i.Start.Equal(i.End) || i.Start.After(i.End)` is equivalent to `if !i.Start.Before(i.End)`

Reply:
Ahaha yup.  Done.